<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreFermeture</class>
 <widget class="QDialog" name="FenetreFermeture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fermeture volleyclubstats</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Sauvegarder</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Continuer plus tard</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
